tools/opt/opt.cpp

```diff
@@ -61,6 +61,8 @@ NOTE: The optimizer is a work in progress.
                e.g.: --set-spec-const-default-value "1:100 2:400"
   --unify-const
                Remove the duplicated constants.
+  --inline
+               Exhaustively inline all function calls
```

Comment:
Description is wrong:  It only inlines entry points.

Also, let's avoid confusion with LLVM's opt command.  There "-inline" means something else.
I prefer to rename this option  "--inline-entry-points" or even "--inline-entry-points-all"